README.md

```diff
@@ -11,316 +11,264 @@
+Optional conditions can be added to each check, to automatically validate whether it passed or not.
+--->
+
+## ⏩  Getting Started
+
+<details close>
+   <summary>
+      <h3>
+         💻 Installation
+      </h3>
+   </summary>
+
+#### Deepchecks Testing (and CI) Installation
 
 ```bash
 pip install deepchecks -U --user
 ```
 
+For installing also the nlp submodle, replace ``deepchecks`` with ``"deepchecks[vision]"``
+and for installing it with the computer vision submodule, replace it with ``"deepchecks[nlp]"``. 
+For installing with conda, similarly use: ``conda install -c conda-forge deepchecks``.
 
-> Note: Vision & NLP Install
->
-> To install deepchecks together with the **Computer Vision Submodule** that 
-> is currently in *beta release*, replace 
-> ``deepchecks`` with ``"deepchecks[vision]"`` as follows:   
-> ```bash
-> pip install "deepchecks[vision]" -U --user
-> ```
->  
-> To install deepchecks together with the **NLP Submodule** that 
-> is currently in *beta release*, replace 
-> ``deepchecks`` with ``"deepchecks[nlp]"`` as follows:   
-> ```bash
-> pip install "deepchecks[nlp]" -U --user
-> ```
->  
-   
-### Using conda
-
-```bash
-conda install -c conda-forge deepchecks
-```
-
-
-
-## ⏩ Try it Out!
-
-### 🏃‍♀️ See It in Action
-
-Head over to one of our following quickstart tutorials, and have deepchecks running on your environment in less than 5 min:
+Check out the full installation instructions for deepchecks testing [here](https://docs.deepchecks.com/stable/getting-started/installation.html).
 
-- [Train-Test Validation Quickstart (loans data)](
-   https://docs.deepchecks.com/stable/user-guide/tabular/auto_quickstarts/plot_quick_train_test_validation.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out)
-- [Data Integrity Quickstart (avocado sales data)](
-   https://docs.deepchecks.com/stable/user-guide/tabular/auto_quickstarts/plot_quick_data_integrity.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out)
-- [Model Evaluation Quickstart (wine quality data)](
-   https://docs.deepchecks.com/stable/user-guide/tabular/auto_quickstarts/plot_quickstart_in_5_minutes.html?utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out)
+#### Deepchecks Monitoring Installation
 
-> **Recommended - download the code and run it locally** on the built-in dataset and (optional) model, or **replace them with your own**.
-
-
-### 🚀 See Our Checks Demo
-
-Play with some of the existing checks in our [Interactive Checks Demo](
-   https://checks-demo.deepchecks.com/?check=No+check+selected&utm_source=github.com&utm_medium=referral&utm_campaign=readme&utm_content=try_it_out), 
-and see how they work on various datasets with custom corruptions injected.
+If you're using deepchecks also for production monitoring,
+you can deploy a hobby instance in one line on Linux/MacOS (Windows is WIP!) with Docker:
```

Comment:
what is a hobby instance?

Reply:
hinting that it's not "production grade" and if you need something that's scalable, and you care about production-related metrics (e.g. downtime) then you should consider to use the managed/licensed deployment

Reply:
phrasing inspired by posthog, but I think it's used widely enough

Reply:
I was a bit baffled by it, maybe "free tier" or something?

Reply:
updated :) thanks